docs/framework/react/guides/optimistic-updates.md

```diff
@@ -121,8 +121,8 @@ useMutation({
     queryClient.setQueryData(['todos'], context.previousTodos)
   },
   // Always refetch after error or success:
-  onSettled: () => {
-    queryClient.invalidateQueries({ queryKey: ['todos'] })
+  onSettled: async () => {
+    return await queryClient.invalidateQueries({ queryKey: ['todos'] })
   },
```

Comment:
I see we weren’t returning the promise before, so yes, we can do that, but we should likely use the same syntax as above:

```
onSettled: () => queryClient.invalidateQueries({ queryKey: ['todos'] })
```